lib/xcpretty/term.rb

```diff
@@ -0,0 +1,26 @@
+
+module XCPretty
+  class Term
+
+    COLOR_DETECTION_COMMAND = "tput colors"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.